src/test/java/pl/szczesniak/dominik/whattowatch/users/domain/model/UserServiceInFileTest.java

```diff
@@ -0,0 +1,23 @@
+package pl.szczesniak.dominik.whattowatch.users.domain.model;
+
+import org.junit.jupiter.api.Test;
+import pl.szczesniak.dominik.whattowatch.users.infrastructure.persistence.InFileUserRepository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class UserServiceInFileTest {
```

Comment:
To jest test integracyjny UserService z wykorzystaniem repozytorium opartym na pliku. Zamiast tego lepiej zrobić test jednostkowy UserService oparty o InMemoryRepo + dodatkowo test integracyjny samego InFileUserRepository